config/index.js

```diff
@@ -17,6 +17,20 @@ var conf = convict({
     format: [ 'dev', 'test', 'stage', 'prod' ],
     env: 'NODE_ENV'
   },
+  geodb: {
+    dbPath: {
+      doc: 'Path to the maxmind database file',
+      default: path.resolve(__dirname, '../node_modules/fxa-geodb/db/cities-db.mmdb'),
```

Comment:
@jrgm - will you need a puppet config update so that you can specify where the database lives?